clueless/models.py

```diff
@@ -226,6 +226,7 @@ class Game(models.Model):
     startTime = models.DateTimeField(default = datetime.datetime.now, blank = True)
     lastUpdateTime = models.DateTimeField(default=datetime.datetime.now, blank=True)
     hostPlayer = models.ForeignKey(Player)
+    name = models.CharField(max_length=60)
 
```

Comment:
When you run make migrations it should allow you to provide a temporary default- you can fill it with "test" or something innocuous.  Alternative is to nuke the database before running make migrations as well.